apps/src/netsim/NetSimPulseRateControl.js

```diff
@@ -45,10 +45,11 @@ NetSimPulseRateControl.inherits(NetSimSlider.DecimalPrecisionSlider);
  * @override
  */
 NetSimPulseRateControl.prototype.valueToLabel = function (val) {
-  if (val === 1) {
-    return i18n.xSecondPerPulse({ x: val });
+  var rounded = Math.floor(val * 100) / 100;
+  if (rounded === 1) {
+    return i18n.xSecondPerPulse({ x: rounded });
```

Comment:
Just FYI: some languages actually have more cardinalities than "one" and "many" (e.g. Polish, see http://www.unicode.org/cldr/charts/25/supplemental/language_plural_rules.html), however supporting this is not in scope for any of our products according to Tanya.